### DISCUSSION TITLE
The current exchange rate for BTC-EUR is not displayed. (Manual market data)

### DISCUSSION THREAD

**Discussion by Krabtus** [ghostfolio/ghostfolio#3620] [Q&A]
Hello everyone,

I have a problem with Ghostfolio and hope someone can help me. I manually added market data for the Bitcoin-EUR rate as Ghostfolio doesn't seem to be able to retrieve EUR values ​​of Bitcoin directly via Coingecko.

The details of my configuration are as follows:

Symbol: BTC-EUR
Currency: EUR
Data source: Manual
Name: Bitcoin
Currency: EUR
Asset class: Liquidity
Asset subclass: Cryptocurrency
Scraper configuration:
```
{
  "selector": "$.chart.result[0].meta.regularMarketPrice",
  "url": "https://query1.finance.yahoo.com/v8/finance/chart/BTC-EUR"
}
```
URL: https://query1.finance.yahoo.com/v8/finance/chart/BTC-EUR

The problem is that the current EUR value of the Bitcoin holdings is not displayed in Ghostfolio, although testing the scraper configuration shows the correct price value. What could be the reason why the current price value does not appear in Ghostfolio even though the scraper configuration seems to be correct? Could there be a step I missed?

I would appreciate any help or pointers that might help resolve this issue.

Thanks in advance!

**Comment by Krabtus** [comment]
Additionally, I would like to clarify that the market data field for the statistics is updated at night, but not continuously during the day. Therefore, I am not seeing the current value of the inventory. How can I make sure that the current value is always shown?

I would appreciate any help or advice that could solve this problem.

Thanks in advance!

**Reply by dtslvr** [reply]
Related issue: https://github.com/ghostfolio/ghostfolio/issues/3645
